packages/kbn-rule-data-utils/technical_field_names/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "main": "../target_node/technical_field_names",
+  "types": "../target_types/technical_field_names"
+}
```

Comment:
nit: surprised your IDE/prettier isn't adding new lines here.

Reply:
That's what's defined in the `.editorconfig` file, which is automatically managed by the EditorConfig plugin in IDEs https://github.com/elastic/kibana/blob/main/.editorconfig#L12-L13